packages/utilities/src/Customizer.tsx

```diff
@@ -56,7 +56,13 @@ export type ICustomizerProps = IBaseProps &
      * ```
      */
     scopedSettings: Settings | SettingsFunction;
-  }>;
+  }> & {
+    /**
+     * Optional transform function for context. Any implementations should take care to return context without
+     * mutating it.
+     */
+    contextPlugin?: (context: Readonly<ICustomizerContext>) => ICustomizerContext;
```

Comment:
contextTransform?

Reply:
Yeah I think I agree with that name change.